packages/react-dev-utils/README.md

```diff
@@ -38,20 +38,20 @@ module.exports = {
   },
   // ...
   plugins: [
-    // Makes the public URL available as %PUBLIC_URL% in index.html, e.g.:
-    // <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    new InterpolateHtmlPlugin({
-      PUBLIC_URL: publicUrl
-      // You can pass any key-value pairs, this was just an example.
-      // WHATEVER: 42 will replace %WHATEVER% with 42 in index.html.
-    }),
     // Generates an `index.html` file with the <script> injected.
     new HtmlWebpackPlugin({
       inject: true,
       template: path.resolve('public/index.html'),
     }),
     // ...
   ],
+  // Makes the public URL available as %PUBLIC_URL% in index.html, e.g.:
```

Comment:
Moved below  HtmlWebpackPlugin otherwise the tapable hooks are not available for tapping

Reply:
This is down one line too far outside the closing bracket for `plugins:[...]`

Reply:
@thephw thanks, fixed!